src/AstroBase.jl

```diff
@@ -115,4 +115,64 @@ function tio_locator(jd1, jd2)
     -47e-6 * t * sec2rad(1)
 end
 
+"""
+    precession_rate_part_of_nutation(jd1, jd2)
+
+Returns precession corrections for a given 2 part Julian date (TT).
+
+# Example
+
+```jldoctest
+
```

Comment:
Are you going to add the example later? :-)

Reply:
yes! I'll add it later :smile:

Reply:
How about now? 😉 Then I can merge this...